draft-ietf-httpbis-message-signatures.md

```diff
@@ -1048,6 +1048,47 @@ Signature: sig1=:P0wLUszWQjoi54udOtydf9IWTfNhy+r53jGFj9XZuP4uKwxyJo\
 
 The proxy's signature and the client's original signature can be verified independently for the same message, depending on the needs of the application.
 
+# Requesting Signatures
+
+While a signer is free to attach a signature to a request or response without prompting, it is often desirable for a potential verifier to signal that it expects a signature from a potential signer using the `Accept-Signature` header.
+
+The message to which the requested signature is applied is known as the "target message". When the `Accept-Signature` header is sent in an HTTP Request message, the header indicates that the client desires the server to sign the response using the identified parameters. When used in a response, the header indicates that the server desires the client to sign its next request to the server with the identified parameters.
```

Comment:
Is it just limited to the next request (e.g., after redirection if it occurs in a 3xx) or is it _subsequent_ requests?

Reply:
That's a good question. My thought was that it was the next request, not all subsequent requests. Unlike authentication, I don't think I'd necessarily expect signatures to be applied all the time for a given service. I'd like to hear more voices on that though because I could be limited by my own experience.